Source/Urho3D/CSharp/Managed/Scene/Node.cs

```diff
@@ -20,23 +20,44 @@
 // THE SOFTWARE.
 //
 
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
 namespace Urho3DNet
 {
     public partial class Node
     {
-        public T CreateComponent<T>(CreateMode mode = CreateMode.Replicated, uint id = 0) where T: Component
-        {
+        public T CreateComponent<T>(CreateMode mode = CreateMode.Replicated, uint id = 0) where T : Component
+        {            
```

Comment:
No trailing spaces please.

Reply:
VS auto-format is now disabled, if you have a predefined style for format automation that would be nice.